machine-learning/04-medulloblastoma_LV_differences.Rmd

```diff
@@ -84,6 +87,7 @@ plier_file <- file.path(model_dir, "GSE37382_PLIER_model.RDS")
 **Output**
 
 ```{r}
+# TODO: change filename
```

Comment:
You didn't need to change the filename then?

Reply:
Good catch! 🦅 👀

Reply:
Updated in https://github.com/AlexsLemonade/training-modules/pull/39/commits/7425a94005c2c13b9e75b7356b018a62279436d1